pkg/sql/executor.go

```diff
@@ -56,22 +57,25 @@ var errNotRetriable = errors.New("the transaction is not in a retriable state")
 
 const sqlTxnName string = "sql txn"
 const sqlImplicitTxnName string = "sql txn implicit"
+const defaultMetricsSampleInterval = 10 * time.Second
 
 // Fully-qualified names for metrics.
 var (
-	MetaLatency       = metric.Metadata{Name: "sql.latency"}
-	MetaTxnBegin      = metric.Metadata{Name: "sql.txn.begin.count"}
-	MetaTxnCommit     = metric.Metadata{Name: "sql.txn.commit.count"}
-	MetaTxnAbort      = metric.Metadata{Name: "sql.txn.abort.count"}
-	MetaTxnRollback   = metric.Metadata{Name: "sql.txn.rollback.count"}
-	MetaSelect        = metric.Metadata{Name: "sql.select.count"}
-	MetaDistSQLSelect = metric.Metadata{Name: "sql.distsql.select.count"}
-	MetaUpdate        = metric.Metadata{Name: "sql.update.count"}
-	MetaInsert        = metric.Metadata{Name: "sql.insert.count"}
-	MetaDelete        = metric.Metadata{Name: "sql.delete.count"}
-	MetaDdl           = metric.Metadata{Name: "sql.ddl.count"}
-	MetaMisc          = metric.Metadata{Name: "sql.misc.count"}
-	MetaQuery         = metric.Metadata{Name: "sql.query.count"}
+	MetaLatency            = metric.Metadata{Name: "sql.latency"}
+	MetaTxnBegin           = metric.Metadata{Name: "sql.txn.begin.count"}
+	MetaTxnCommit          = metric.Metadata{Name: "sql.txn.commit.count"}
+	MetaTxnAbort           = metric.Metadata{Name: "sql.txn.abort.count"}
+	MetaTxnRollback        = metric.Metadata{Name: "sql.txn.rollback.count"}
+	MetaSelect             = metric.Metadata{Name: "sql.select.count"}
+	MetaDistSQLSelect      = metric.Metadata{Name: "sql.distsql.select.count"}
+	MetaDistSQLExecLatency = metric.Metadata{Name: "sql.distsql.exec.latency"}
```

Comment:
These have been added in between the the two stats they are mimicking, can you move this pair below MetaSQLExecLatency?